# Assignment \#6 : Nuclear Reactions

Kaimi Kahihikolo

Due: 25 March 2019

In [1]:
import astropy.constants as const
import astropy.units as u
u.imperial.enable();

___
## (1) Average Energy Generation Rate

- **(1.1)** Calculate the energy generation rate per unit mass in the Sun, in ergs per gram per second. 

- **(1.2)** Compare with the case of a human being with a mass of 80 kg producing 2000 kilocalories per day. What is a calorie? Check in the section about units in Astrophysical Quantities.

- **(1.3)** Compare also with the average rate of energy production per unit mass in a typical car.
___

### (1.1) For the Sun...
Given that $M_\odot \simeq 1.9884754 \times 10^{33} \, [g]$ and $L_\odot \simeq 3.828 \times 10^{33} \, [erg \, s^{-1}]$. The energy generation rate per unit mass in the sun is,

$$q = \frac{L_\odot}{M_\odot}$$

which is

In [2]:
P_sun = const.L_sun.to(u.erg/u.s)
M_sun = const.M_sun.to(u.g)

(P_sun/M_sun).round(2)

<Quantity 1.93 erg / (g s)>

### (1.2) For the Average Human...

The average human produces 2000 kilocalories per day, which is equivalent to $4184 \, [J]$ per day or $4.184 \times 10^{10} \, [erg]$ per day. If the mass of an average human is $80 \, [kg]$, then the average energy generation rate is,

In [3]:
P_human = (2000. * u.imperial.kcal / u.day).to(u.erg/u.s)
M_human = (80. * u.kg).to(u.g)

(P_human / M_human).round(2)

<Quantity 12106.48 erg / (g s)>

### (1.3) For the Average Car...

Take an average car, for example the Honda civic, which has a horsepower of about 180 hp and weighs approximately 3000 lbs. $180 \, [hp] \simeq 1.34 \times 10^{12} \, [erg\, s^{-1}]$ and $3000 \, [lbs]$ $\simeq$ $1360777 \, [g]$.

In [4]:
M_car = (3000. * u.imperial.lb).to(u.g)
P_car = (180. *u.imperial.hp).to(u.erg/u.s)

(P_car / M_car).round(2)

<Quantity 986392.08 erg / (g s)>

In summary,

| Honda Civic | Average Human | Our Sun | Units |
| :---: | :---: | :---: | ---: |
| $9.8\times 10^{7}$ | $1.21\times 10^{4}$ | $1.93$ | $\frac{erg}{g\,s}$ |

___
## (2) Energy Production

- **(2.1)** Using the table of atomic mass excesses, calculate the energy in MeV generated in the production of an alpha particle (He nucleus) by the fusion of 4 protons in the solar core.

- **(2.2)** What mass of He per second must be generated to produce the solar luminosity?

- **(2.3)** Finally, calculate the energy in MeV generated in the production of a ${}^{12} C$ nucleus by the fusion of 3 alpha particles.
___

### (2.1) Producing an Alpha Particle (He Nucleus)

The reaction of interest is, $4 H' \rightarrow He^{4} + E$, where $E$ is the excess energy. We can calculate the energy in this process from the mass excessess. Recall that $\Delta m \, H' = 7.28899$ and $\Delta m \, He^{4} = 2.42475$.

Thus,

$$\Delta m \, H' \cdot 4 - \Delta m \, He^{4} = $$

In [5]:
H = 7.28899 * u.MeV
He4 = 2.42475 *u.MeV

H_He = (4*H - He4)
H_He

<Quantity 26.73121 MeV>

Thus the energy produced is about $26.73 \, [MeV]$.

### (2.2) Solar Luminosity

If we compare the ratio of $L_\odot : \text{"H_He"}$, we can arrive at an estimate for the average reactions per second.

In [6]:
Rate = (const.L_sun / H_He).decompose()
Rate

<Quantity 8.93805291e+37 1 / s>

If we multiply this rate by $2.42475$,

In [7]:
(Rate * He4)

<Quantity 2.16725438e+38 MeV / s>

When expressed in terms of grams, is approximately $6 \times 10^{14} \, [g \, s^{-1}]$.

### (2.3) C$^{12}$ Production

We now want to calculate the energy production of $3 He^{4} \rightarrow C^{12}$. If we recall that $\Delta m \, He = 2.42475$ and $\Delta m \, C^{12} = 0$.

$$\Delta m \, He^{4} \cdot 3 - 0 = $$

In [8]:
3 * He4

<Quantity 7.27425 MeV>

Thus, the energy generated in the production of ${}^{12} C$ is approximately $7.27 \, [MeV]$